### PR TITLE
TSK-1356: added cypress e2e testing

### PR DESCRIPTION
**Pull request by Tristan2357**
Added Cypress and some e2e tests. These tests run during the "WEB" job in travis, but could be split into their own stage.
Our ldifs have been modified, because our sample data contained invalid owners in the workbaskets.

https://sonarcloud.io/dashboard?branch=TSK-1356&id=Tristan2357_taskana

Documentation
[Taskana Web Application](https://taskana.atlassian.net/l/c/1C7T1XAm)
[End-to-End Testing with Cypress](https://taskana.atlassian.net/l/c/FTc9Qaq5)

### CODE REVIEW

File: ci/cypress.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (tbdn) [inline ci/cypress.json:1]: Do we need 3 versions of this file?

File: web/cypress.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (tbdn) [inline web/cypress.json:1]: same as web/cypress/cypress.json

File: web/cypress/cypress.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (tbdn) [inline web/cypress/cypress.json:1]: same as web/cypress.json

File: ci/cypress.json (hunk 1 above)
Comment (Tristan2357) [reply ci/cypress.json:1]: No just 2, one for local testing and the other one, for travis (isLocal flag and the videos/screenshots)

File: web/cypress/cypress.json (hunk 3 above)
Comment (Tristan2357) [reply web/cypress/cypress.json]: i removed this one

**Pull request merged**